docs/en/setup/backend/backend-alarm.md

```diff
@@ -180,6 +180,23 @@ slackHooks:
     - https://hooks.slack.com/services/x/y/z
 ```
 
+## WeChat Hook
+Notice: This is only apply to Enterprise Webchat. To use Wechat webhook you need to follow the [Enterprise Wechat Webhooks guide](https://work.weixin.qq.com/help?doc_id=13376).
```

Comment:
Take a look at my previous comment. This seems not called enterprise wechat.

Reply:
Ok, thanks